src/FakeItEasy/Creation/CastleDynamicProxy/CastleDynamicProxyInterceptionValidator.cs

```diff
@@ -34,7 +34,7 @@ private static string GetReasonForWhyMethodCanNotBeIntercepted(MethodInfo method
                 var explicitImplementation = method.Name.Contains('.');
                 if (!explicitImplementation)
                 {
-                    return "Sealed methods can not be intercepted.";
+                    return "Method should be marked as virtual, abstract or part of an Interface in oreder to be intercepted.";

```

Comment:
@chrismatheson,

1. Congratulations!
2. I hope you didn't drop the child on the floor! (sorry)
3. No rush at all. We're in no hurry. We just wanted to gauge your interest. Now that we know for sure, we won't hound you for at least a couple of weeks. :wink:

Thanks.